go/agents/sail-researcher/.env.example

```
@@ -0,0 +1,9 @@
+# Required
+GOOGLE_CLOUD_PROJECT=your-google-cloud-project-id
+GOOGLE_MAPS_KEY=your-google-maps-api-key
+GEMINI_API_KEY=your-gemini-api-key
+
+# Optional (defaults shown)
+ENV=development
+PORT=8081
+MODEL=gemini-2.5-flash
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MODEL key should go before the PORT key

(UnorderedKey)